packages/dds/cell/src/cell.ts

```diff
@@ -43,6 +43,42 @@ interface ICellValue {
     value: unknown;
 }
 
+// TODO: this should reference a shared interface in @fluidframework/runtime-definitions so it's usable from
+// here and @fluidframework/attributor
+interface AttributionInfo {
+    /**
+     * The user that performed the change.
+     */
+    user: string;
+    /**
+     * When the change happened.
+     */
+    timestamp: number;
+}
+
+/**
+ * {@inheritDoc CellAttributor}
+ */
+export class CellAttributor {
```

Comment:
i'm not sure a separate object here is necessary. we're basically just storing a single piece of state.